coalib/bears/Bear.py

```diff
@@ -461,13 +461,18 @@ def execute(self, *args, debug=False, **kwargs):
                          .format(name), str(exc))
 
             if self.kind() == BEAR_KIND.LOCAL:
```

Comment:
you can add an inline logic here instead of adding extra code.

Reply:
but adding inline logic will increase the length of line to be more than 80 chars

Reply:
```python
if self.kind() == BEAR_KIND.LOCAL and self.section['log_level'].value != 'DEBUG':
```

Reply:
The benefit is that you don't need to do changes unrelated to the real problem.

Reply:
no problem @li-boxuan you are helping me that is in fact great thanks